dockerfiles/rocky/is/docker-entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2022 WSO2, LLC. (http://wso2.com)
```

Comment:
update license header

Reply:
Addressed with https://github.com/wso2/docker-is/pull/418/commits/0db06b1a24e55a428dcdc4f6621fdf1fc9f314fc